### ISSUE TITLE
TypeError: object of type 'NoneType' has no len() targeting Python

### ISSUE THREAD

**Issue by supergeoff** [haxetink/tink_cli#15]
Hello,
I came across an issue while doing some tests using Tink_Cli.
See following code:
```haxe
import tink.cli.*;
import tink.Cli;

class Main {
	/**
	 * REQUIRED - Specify a first flag
	 */
	@:required
	public var flag:String;

	/**
	 * Specify a second flag
	 */
	public var anotherflag:String;

	static function main() {
		Cli.process(Sys.args(), new Main()).handle(Cli.exit);
	}

	public function new() {}

	@:defaultCommand @:skipFlags
	public function default_command() {
		Sys.println(Cli.getDoc(this));
	}

	/**
	 * Show Test #1
	 */
	@:command
	public function test1(rest:Rest<String>) {
		trace(flag);
	}

	/**
	 * Show Test #2
	 */
	@:command @:skipFlags
	public function test2() {
		trace("Test #2");
	}

	/**
	 * Write a file
	 */
	@:command @:skipFlags
	public function write() {
		trace("\n[INFO] Writing to file...");
	}
}
```
This execute properly, but if you remove the digit from the function test1 or test2 or if you rename them without a digit in the name, then the execution in Python without args (should retrieve the help command as it is the default command) result in the following error:

`Traceback (most recent call last):
  File "Main.py", line 2245, in <module>
    Main.main()
  File "Main.py", line 131, in main
    tink_cli_Router0(Main(),tink_cli_prompt_RetryPrompt(5)).process(Sys.args()).handle(tink_Cli.exit)
  File "Main.py", line 1280, in process
    _hx_local_0 = len(_g)
TypeError: object of type 'NoneType' has no len()
`

Same is true if you keep the name unchanged while forcing it through metadata:
```haxe
@:command("test")
public function test1(rest:Rest<String>) {
	trace(flag);
}
```

But calling the program while specifying the flag "test" does work. Thus:
`python main.py test`
Work as intended.

Best regards;

**Comment by kevinresol** [comment]
Closed by 11690cfff22c4270e2b9d51e232ca281c65e73f2

**Comment by supergeoff** [comment]
Thank you sir :) Works as intended now !
